frontend/src/modules/lf/segments/components/fragments/lf-project-count.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <div class="min-w-[105px]">
+    <lf-pill color="transparent" type="bordered" class="!flex items-center gap-1 w-full">
```

Comment:
This is not following the design. On the Status chips/pills, the width is fixed. But I don't think it's the case for this one, might be worth checking with Nuno. Also, "Project" shouldn't be capitalized
![Screenshot 2024-12-04 at 08 18 51](https://github.com/user-attachments/assets/50bb204d-ce08-45a0-82b3-1e4ea8145476)